atomic_reactor/plugins/pre_inject_parent_image.py

```diff
@@ -69,6 +70,7 @@ def run(self):
         self.find_repositories()
         self.select_new_parent_image()
         self.validate_new_parent_image()
+        self.adjust_new_parent_image()
```

Comment:
Don't we want to adjust before we validate it?

Reply:
yeah we do :) moved